docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,433 @@
+      "text/plain": [
+       "<Figure size 1434x294.311 with 1 Axes>"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.library import EfficientSU2\n",
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "from qiskit_ibm_runtime import SamplerV2\n",
+    "from qiskit_aer.primitives import EstimatorV2\n",
+    "\n",
+    "from qiskit_addon_cutting import (\n",
+    "    cut_gates,\n",
+    "    generate_cutting_experiments,\n",
+    "    reconstruct_expectation_values,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "circuit = EfficientSU2(num_qubits=4, entanglement=\"circular\").decompose()\n",
+    "circuit.assign_parameters([0.4] * len(circuit.parameters), inplace=True)\n",
+    "\n",
+    "\n",
+    "observable = SparsePauliOp([\"ZZII\", \"IZZI\", \"-IIZZ\", \"XIXI\", \"ZIZZ\", \"IXIX\"])\n",
+    "print(f\"Observable: {observable}\")\n",
+    "circuit.draw(\"mpl\", scale=0.8)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ef29b75-e790-4f3a-b2f7-4a18928cf5e6",
+   "metadata": {},
+   "source": [
+    "Each of the [`CNOT`](../api/qiskit/qiskit.circuit.library.CXGate) gates between qubits $q_0$ and $q_3$ will introduce two swap gates after transpilation (assuming the qubits are connected together in a straight line). To avoid this increase in depth, we can replace these distant gates with [`TwoQubitQPDGate`s](../api/qiskit-addon-cutting/qpd-two-qubit-qpd-gate) using the [`cut_gates()`](../api/qiskit-addon-cutting/qiskit-addon-cutting#cut_gates) method.  This function will also return a list of [`QPDBasis`](../api/qiskit-addon-cutting/qpd-qpd-basis) instances - one for each decomposition."
```

Comment:
```suggestion
    "Each of the [`CNOT`](/api/qiskit/qiskit.circuit.library.CXGate) gates between qubits $q_0$ and $q_3$ introduce two swap gates after transpilation (assuming the qubits are connected in a straight line). To avoid this increase in depth, you can replace these distant gates with [`TwoQubitQPDGate`s](/api/qiskit-addon-cutting/qpd-two-qubit-qpd-gate) using the [`cut_gates()`](/api/qiskit-addon-cutting/qiskit-addon-cutting#cut_gates) method.  This function also returns a list of [`QPDBasis`](../api/qiskit-addon-cutting/qpd-qpd-basis) instances - one for each decomposition."
```